viewer/templates/viewer/help.html

```diff
@@ -6,17 +6,23 @@
   </a>
 </nav>
 
-<h2 class="o-header_alt">Common web page index searches</h2>
+<h2 class="o-header_alt">Common Searches</h2>
 
 <p>
-  The web page index stores a nightly snapshot of webpage data that you can
-  search for all sorts of useful things. You can search for single terms or
-  exact phrases.
+  This tool stores a nightly snapshot of consumerfinance.gov web page data that
+  you can search for all sorts of useful things. Common search examples are
+  below. You can also
+  <a href="">learn more about the data</a>. View results in the browser, or
+  export them as a CSV. For those less handy with Excel, it may be useful to
+  <a
+    href="https://github.cfpb.gov/CFPB/el-camino/issues/642#issuecomment-318309"
+    >learn how to filter our results with query strings</a
+  >
 </p>
 
 <div class="block">
   <h2>Title</h2>
-  <p>Search page titles.</p>
+  <p>Find all instances of a string in the title of a cf.gov page.</p>
   <p class="u-bold_text">Examples</p>
```

Comment:
It's a little hard to tell from the design, but is this really supposed to be bolder than the h2? The h2 is weight 400, and this is weight 700.

Reply:
According to the design, this is intended